examples/README.md

```diff
@@ -214,6 +214,32 @@ process2.wait()
 )
 ```
 
+## Stream from a local video to HTTP server
+
+```python
+video_format = "flv"
+server_url = "http://localhost:8080"
```

Comment:
You should set an IP address for the listening address, such as `127.0.0.1`

Reply:
But, using `server_url = "http://localhost:8080"` is equal to using `server_url = "http://127.0.0.1:8080"`. Both work, right?

Reply:
You shouldn't do a DNS lookup for listening to a socket (even if the host in question is in /etc/hosts) if that's not specifically intended, and I don't think it is.